src/functions/inherits.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function inherits(ctor, superCtor) {
+  ctor.prototype = Object.create(superCtor.prototype, {
```

Comment:
Sure the two are absolutely equivalent? I would have replace it with the implementation of [`util.inherits`](https://github.com/nodejs/node/blob/29867f35d89c9cc0ef1ea7dfdecd6361dc00a7c1/lib/util.js#L133-L151) (minus the error validation).

Reply:
Looks like the node implementation changed, however I used the one from `inherits` (which is used in the `util` npm package): 

https://github.com/isaacs/inherits/blob/f721d6bee2d6df13262a190fee1f48e21ed72b42/inherits_browser.js#L3-L13

Reply:
that's likely because Object.setPrototypeOf is IE11+ only